parachain/src/command.rs

```diff
@@ -380,6 +380,14 @@ pub fn run() -> Result<()> {
 
 					Err("Chain runtime doesn't support benchmarking".into())
 				}),
+				// https://github.com/paritytech/substrate/blob/1802a115e8480fd7a4654d45c85b58c2189c508a/client/db/src/lib.rs#L1096
+				#[cfg(feature = "runtime-benchmarks")]
+					BenchmarkCmd::Storage(_) => Err(
+						"Storage benchmarking can be enabled with `--features runtime-benchmarks`."
```

Comment:
Why do we need to disable storage benchmarking? I can't see the reason at the link above.

Could we change the error message to mention that storage benchmarking is disabled? This will only be displayed when the runtime-benchmarks feature is enabled, but this message advises enabling the same feature.

```suggestion
						"Storage benchmarking is disabled."
```

Reply:
I missed the `not` clause here, corrected now. https://github.com/Snowfork/snowbridge/pull/710/commits/8fc8037d9d7317a718d93bd8e945d771c5a3da47